expo/gbe/functions.py

```diff
@@ -97,6 +101,10 @@ def get_conference_days(conference):
     return conference.conferenceday_set.all()
 
 
+def get_conference_days_slug(slug):
+    return Conference.by_slug(slug).conferenceday_set.all()
+
+
```

Comment:
Sure.  Wrote it to solve a problem that I may return to, depending upon how cal_times in event_info works.  Then decided that fixing things outside of my code was beyond scope, and forgot about it.  Will pull the code and drop it into a comment in the apropos ticket.